tools/sof-kernel-log-check.sh

```diff
@@ -345,6 +345,15 @@ ignore_str="$ignore_str"'|iwlwifi [[:digit:].:]+: '
 # SDW related logs
 #
 
+#Irrelated log brought in verify-kernel-boot-log.sh
+#http://sof-ci.sh.intel.com/#/result/planresultdetail/4120?model=CML_RVP_HDA&testcase=verify-kernel-boot-log
+ignore_str="$ignore_str"'|Failed to mount NFSD configuration filesystem'
+#http://sof-ci.sh.intel.com/#/result/planresultdetail/3937?model=TGLH_0A70_HDA&testcase=verify-kernel-boot-log
+ignore_str="$ignore_str"'|DMAR: DRHD: handling fault status reg '
+#http://sof-ci.sh.intel.com/#/result/planresultdetail/3937?model=CML_SKU0983_SDW&testcase=verify-kernel-boot-log
+ignore_str="$ignore_str"'|tpm tpm0: tpm_try_transmit: send(): error'
```

Comment:
Please first try to:
- disable TPM in the BIOS
- blacklisting `tpm_crb` and `tpm_tis`

Reply:
sure, let me check the BIOS setting

Reply:
I disabled TPM from the BIOS. this error goes away.
Security->TPM 2.0 Security
![20210526_231229_tpm](https://user-images.githubusercontent.com/45636982/119840157-92706a00-beb9-11eb-8b7c-8092598a1705.jpg)

Reply:
@fredoh9 Thanks so much fred :)